cicd/builds/triggering-builds-build-hooks.adoc

```diff
@@ -18,6 +18,13 @@ include::modules/builds-using-gitlab-webhooks.adoc[leveloffset=+3]
 
 include::modules/builds-using-bitbucket-webhooks.adoc[leveloffset=+3]
 
+include::modules/unauthenticated-users-system-webhook.adoc[leveloffset=+3]
```

Comment:
We should make it as a pre-requisite o post-requisite step to highlight that this must be done for the feature to work.